entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# set -x
+
+# run the testing
+# coverage run --source=. -m unittest "$*"
+python3 -m pytest -v --html=/output/report.html --self-contained-html
```

Comment:
This looks good. Recommend adding an optional way to run a single file or test (rather than the whole test suite) for faster feedback when developing a new processor. I believe this can be done by passing a positional argument between `pytest` and `-v`.

_Here is what I am looking at: https://stackoverflow.com/questions/34833327/how-to-test-single-file-under-pytest_

Reply:
Implemented based on work @jeffsuperglide had done.  Maybe need future refinements, but default ./tests.sh will run non verbose tests for everything (the intended GH action preference).

Reply:
I see this in the latest commit

`python3 -m pytest --html=/output/report.html --self-contained-html "$*"`

Resolving